### ISSUE TITLE
Unclear instructions on how to configure 

### ISSUE THREAD

**Issue by shivankitsharma** [kromerm/adflab#19]
![image](https://user-images.githubusercontent.com/37122288/41496198-88202408-70ee-11e8-8b58-ee90f176627f.png)

The instruction are unclear about what to do here. It says for each SQL Server connection configure the connection string, server name, and password. I only see the password field empty so i added the password there. The connection string and server name are already present so do they have to be replaced by the setting of the azure version of the DW, ODS and OLTP from the Azure console?

Further, does adflabstaging need to be replaced by adflabstorage with the corresponding connection string and key?

Please help!

**Comment by kromerm** [comment]
Can you tell me which lab document and page this is from?

**Comment by shivankitsharma** [comment]
Hey Mark, 

Thank you for taking out the time to reply. I had almost given up in trying to figure out what's wrong 

LAb 2 - Page 16 - Instructions 24-27. I added the password for the labadmin account to all the 3 ODS, OLAP and DW connections and made no further changes. For the adflabstaging i don't know what to do because my Azure account is named adflabstorage and there is no staging account. I still added the Accountkey for the adflabstorage account and changed the account name and connection string to those found in the azure interface for the adfstorage account. 
![image](https://user-images.githubusercontent.com/37122288/41504565-f2acfb02-71a7-11e8-918f-79d278cbb1de.png)

When i run it, it fails and throws up a couple of errors

![image](https://user-images.githubusercontent.com/37122288/41504570-17cf234c-71a8-11e8-839e-43fce7e7770f.png)

Further if i tried to continue, i see nointegration runtime options 

![image](https://user-images.githubusercontent.com/37122288/41504584-53cf9d22-71a8-11e8-8b01-b97f20208f4a.png)

Help!

**Comment by kromerm** [comment]
Is the SSIS Integration Runtime present in your Data Factory?

**Comment by shivankitsharma** [comment]
Hey, 

Yes it is and running but seems like it isn't being picked up

![image](https://user-images.githubusercontent.com/37122288/41521822-8e49035e-7288-11e8-9c9a-8e254fc4d4b6.png)

Further, these are the settings in place

![image](https://user-images.githubusercontent.com/37122288/41521969-12c772fa-7289-11e8-9a79-4a8ac790cd8e.png)

**Comment by kromerm** [comment]
@joshuha Is this something you can take a look at?

**Comment by joshuha** [comment]
Yes each ServerName property needs to be updated to the output of your ServerName from the Powershell script. You need to update each connections ServerName and ConnectionString property. Also, if you changed the default admin UserName in the script you would also need to update that.
